model/src/main/java/ru/training/karaf/repo/BookDescriptionRepoImpl.java

```diff
@@ -0,0 +1,75 @@
+package ru.training.karaf.repo;
+
+import org.apache.aries.jpa.template.JpaTemplate;
+import ru.training.karaf.model.*;
+
+import java.util.Collection;
+
+public class BookDescriptionRepoImpl implements BookDescriptionRepo {
+
+    private JpaTemplate template;
+
+
+    public BookDescriptionRepoImpl(JpaTemplate template) {
+        this.template = template;
+    }
+
+    @Override
+    public void create(long bookId, String description, String date) {
+        BookDescriptionDo descriptionToCreate = new BookDescriptionDo();
+
+        //AuthorDo author = template.txExpr(em -> em.createNamedQuery(AuthorDo.GET, AuthorDo.class).setParameter("name", authorName).getSingleResult());
+
+        //descriptionToCreate.setAuthorDos(author);
+        descriptionToCreate.setDescription(description);
+        descriptionToCreate.setDate(date);
+
+        BookDO bookDO = template.txExpr(em -> em.createNamedQuery(BookDO.GET_BY_ID, BookDO.class).setParameter("id", bookId).getSingleResult());
+        descriptionToCreate.setBook(bookDO);
```

Comment:
bookDO might be null and since we are going to make book field not optional for BookDescriptionDO we will have a chance to get an error here.